### PR TITLE
Configurable WMS translation table 

### PR DESCRIPTION
**Pull request by b-Tomas**
### 🗒️ Description

Moved the wms translation table to the YAML configuration file, and the translation itself is now handled in the connector instead of the Instock API.

### CODE REVIEW

File: instock_connector/inorbit_instock_connector/src/connector.py [hunk 1]
```diff
@@ -377,6 +377,16 @@ def _handle_message_action(self, args: list, options: dict):
                 options["result_function"](2)
                 return
 
+            # Translate any items for which the WMS goods code differs from the
+            # article_id stored in InStock
+            _table = self.config.connector_config.wms_translation_table
```
Comment (russell-inorbit) [inline instock_connector/inorbit_instock_connector/src/connector.py:382]: why the leading `_`?

File: instock_connector/config/myinstock.example.yaml [hunk 2]
```diff
@@ -13,3 +13,7 @@ instock-asrs-1:
     instock_site_id: x.y.z-A
     # Position of the ASRS in map space
     pose: { "x": 0, "y": 0, "yaw": 0 }
+    # Map of WMS product IDs to InStock article IDs.
+    wms_translation_table:
+      - R8S1B1-KraftParmesan: R8S1B1  # WMS_CHEESE: INSTOCK_CHEESE
+      - R7S1B1-BarillaPasta: R7S1B1  # WMS_PASTA: INSTOCK_PASTA
```
Comment (russell-inorbit) [inline instock_connector/config/myinstock.example.yaml:19]: I would add this to the internal confg files we have on GitLab and maybe mat this less "robot space" specific in the public repo.

**Review by russell-inorbit — CHANGES_REQUESTED** [review]

File: instock_connector/config/myinstock.example.yaml (hunk 2 above)
Comment (adamantivm) [reply instock_connector/config/myinstock.example.yaml:19]: +1 to make it a config file with sample products that need mapping

**Review by adamantivm — COMMENTED** [review]
nit

**Review by russell-inorbit — APPROVED** [review]

**Comment by russell-inorbit** [comment]
Looks good! Assuming you ran a local test this LGTM.

**Pull request merged**